packages/rulesets/src/oas/functions/oasOpSecurityDefined.ts

```diff
@@ -50,6 +53,31 @@ export default createRulesetFunction<{ paths: Record<string, unknown> }, Options
     const schemes = _get(targetVal, schemesPath);
     const allDefs = isObject(schemes) ? Object.keys(schemes) : [];
 
+    // Check global security requirements
+
+    {
```

Comment:
do we need this block? It doesn't seem like `security` is redeclared within the current scope. It is a later on at https://github.com/stoplightio/spectral/blob/e792e63337288f09b2a33b46cb0b99a256b34f3e/packages/rulesets/src/oas/functions/oasOpSecurityDefined.ts#L84, yet this is another block scope, so we should be good.

Reply:
Amended PR to remove unnecessary block.